Foundation/NSCharacterSet.swift

```diff
@@ -50,13 +50,18 @@ open class NSCharacterSet : NSObject, NSCopying, NSMutableCopying, NSCoding {
     }
     
     open override func isEqual(_ value: Any?) -> Bool {
+        guard let runtimeClass = _CFRuntimeGetClassWithTypeID(CFCharacterSetGetTypeID()) else {
```

Comment:
@phausler oh, and by the way, I'm not sure what to think of:
https://github.com/apple/swift-corelibs-foundation/pull/1092/files#diff-c6672fda2d6b50ba301f8b945cacbd8aR298
Any thoughts?